app/retail/emails.py

```diff
@@ -308,6 +311,7 @@ def render_quarterly_stats(to_email, platform_wide_stats):
     params = {**quarterly_stats, **platform_wide_stats}
     params['profile'] = profile
     params['subscriber'] = get_or_save_email_subscriber(to_email, 'internal'),
+    params['email_type'] = 'roundup'
```

Comment:
should quarterly stats be in the `roundup` category or should it be its own category?

Reply:
cc: @owocki @thelostone-mc ?

Reply:
its own category pls

Reply:
Category name please